distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -949,7 +949,11 @@ public void testCswQueryForOpenGisXml() throws Exception {
         assertThat("Incorrect title", metacard.getTitle(), equalTo(metacardTitle));
         assertThat("Incorrect geometry", "POINT (30.0 10.0)", equalTo(metacard.getLocation()));
 
-        // TODO: THIS ASSERTION FAILS. VERIFY WITH CHRIS IT IS WORKING AS DESIGNED.
+        // TODO: THIS ASSERTION FAILS. SEE DDF-2476 (https://codice.atlassian.net/browse/DDF-2476)
```

Comment:
Updated DDF-2476 with reference to remote the comment and enable the assertion.
<img width="884" alt="screen shot 2017-04-24 at 8 13 50 pm" src="https://cloud.githubusercontent.com/assets/133284/25367550/245624ec-292b-11e7-8937-799d2d7d5a77.png">